src/status_im/contexts/push_notifications/remote_token.cljs

```diff
@@ -0,0 +1,50 @@
+(ns status-im.contexts.push-notifications.remote-token
+  (:require
+    [promesa.core :as promesa]
+    [react-native.firebase :as firebase]
+    [react-native.platform :as platform]
+    [react-native.push-notification-ios :as pn-ios]))
+
+(defn request-remote-token
+  "Request the remote token for the mobile platform.
+   On iOS this will retrieve a APNS token,
+   and on Android this will retrieve a FCM token."
+  [options]
```

Comment:
I might've missed something in the PR, but I've seen many functions receiving and passing around an `options` map, but functions like `start-registration` and others ignore the parameter. Could you elaborate further on why we need for this parameter?

Reply:
I've sorta reserved the first argument to be a parameters map for these functions since I thought it'd be likely we would pass some flags or configuration at some point. Though atm, we currently don't really use it so it could be removed.

Reply:
Leaving here for future reference: we agreed off-GH to keep the `options` parameter and in case it proves to be unnecessary in the future we would try to remove the parameter.